js/gc-script.js

```diff
@@ -19,50 +19,80 @@ panelFontColorPicker.addEventListener("input", function () {
 
 panelBgColorPicker.addEventListener("input", function () {
   panelBgColor.style.backgroundColor = panelBgColorPicker.value;
-  
+
   const darkerShade = tinycolor(panelBgColorPicker.value).darken(5).toString();
-  
+
   accentHeading.forEach((heading) => {
     heading.style.color = darkerShade;
   });
 });
 
 // Function to handle image upload
-document.getElementById("image-upload").addEventListener("change", function (event) {
-  const file = event.target.files[0]; // Get the uploaded file
-  const img = document.getElementById("resume-image");
+document
+  .getElementById("image-upload")
+  .addEventListener("change", function (event) {
+    const file = event.target.files[0]; // Get the uploaded file
+    const img = document.getElementById("resume-image");
+
+    // Check if a file is selected
+    if (file) {
+      const reader = new FileReader();
+
+      // Callback function to set the uploaded image as the src attribute
+      reader.onload = function (event) {
+        img.src = event.target.result;
+      };
+
+      // Read the file as a data URL
+      reader.readAsDataURL(file);
+    }
+  });
 
-  // Check if a file is selected
-  if (file) {
-    const reader = new FileReader();
+function saveResume() {
+  const colPanel = document.getElementById("colPanel");
+  const infoPanel = document.getElementById("infoPanel");
 
-    // Callback function to set the uploaded image as the src attribute
-    reader.onload = function (event) {
-      img.src = event.target.result;
-    };
+  // Change class from col-md-4 to col-4
+  if (colPanel) {
```

Comment:
Instead of changing the classes here, you could make a copy first and then manipulate the classes for the print window only. You can copy the HTML to a new print window and apply your styling on the document from that window. That way you won't have to modify the CSS of the original contents.